spec/support/form_helpers.rb

```diff
@@ -273,6 +274,13 @@ def fill_in_activity_form(
     choose oda_eligibility
     click_button t("form.button.activity.submit")
 
+    if level == "project" || level == "third_party_project"
+      expect(page).to have_content t("form.label.activity.oda_eligibility_lead")
+      expect(page).to have_content t("form.hint.activity.oda_eligibility_lead")
+      fill_in "activity[oda_eligibility_lead]", with: oda_eligibility_lead
+      click_button t("form.button.activity.submit")
+    end
+
```

Comment:
Shall we include an assertion that the content of `oda_eligibility_lead` is displayed on the activity details page? (probably somewhere after line 333).

Reply:
Oh, good shout 👍